package.json

```diff
@@ -1274,6 +1274,42 @@
             "description": "Show regular expressions in completion list.",
             "order": 20
           },
+          "dotnet.backgroundAnalysis.solutionCrawlerBackgroundAnalysisScope": {
+            "type": "string",
+            "enum": [
+              "activeFile",
```

Comment:
@dibarbet Removing active file worked just fine. I have done the change. I'll let @Cosifne answer your other query about renaming the option name completely.

Reply:
Ah, I forget to send my comments.
The general rule for naming is to "match" what's on the server. Like here, if you don't like `SolutionCrawler` then it's better to remove the name from the server.
In theory, we can do whatever we like in the client by creating a hashmap, and mapping from the server name to the client-side name, but it should be avoided when possible.

Reply:
> Like here, if you don't like SolutionCrawler then it's better to remove the name from the server

@Cosifne - wouldn't that cause a breaking change in VS though? Renaming the option on server means now any user on VS who had changed their option value to something else will lose their changed settings.

Reply:
I am pretty sure I have done the rename before. See here https://github.com/dotnet/roslyn/pull/66899
The name of the option is just an internal string, it will be mapped to VS storage [here](https://github.com/dotnet/roslyn/blob/575bc42589145ba18b4f1cc2267d02695f861d8f/src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs#L116) (There is a similar one for VS mac, you can find it by search the name)

These strings would be persisted by VSCode, we can do whatever we want now since it is still in preview.
But once the extension is shipped, it would be hard to change these values.

Reply:
Nice - let me change the server option name then.

Reply:
@Cosifne - https://github.com/dotnet/roslyn/pull/68810